docs/controls/MasterDetailsView.md

```diff
@@ -19,19 +19,34 @@ The [MasterDetailsView Control](https://docs.microsoft.com/dotnet/api/microsoft.
           NoSelectionContentTemplate="{StaticResource NoSelectionContentTemplate}"
           CompactModeThresholdWidth="720">
 </controls:MasterDetailsView>
-```
+```
 
 ## Sample Output
 
 ![MasterDetailsView animation](../resources/images/Controls/MasterDetailsView.gif)
 
+## BackButtonBehavior
+The MasterDetailsView control uses the first Frame in the parent tree to handle moving from the Details view to the Maste view in compact mode. If the host Frame is attempting back navigation while the Details view state is active, the MasterDetailsView will transition to the the Master view and cancel the back navigation.

```

Comment:
This isn't entirely true. No Frame is needed anywhere in the app. The MasterDetailsView does listen to the frame navigation event. If the Frame is navigating back and if the control is in the compact "Details" state, then it will transition to the compact "Master" state.

Also there is a typo on the first sentence
> from the Details view to the **Maste** view